README.md

```diff
@@ -68,12 +68,13 @@ gruntwork-install --module-name 'ecs-scripts' --repo 'https://github.com/gruntwo
 
 ##### Example 2: Download and Install a Script Module with Parameters
 
-Install the [vault-ssh-helper
-module](https://github.com/gruntwork-io/script-modules/tree/master/modules/vault-ssh-helper) from the [script-modules
-repo](https://github.com/gruntwork-io/script-modules), passing two custom parameters to it:
+Install the [fail2ban
+module](https://github.com/gruntwork-io/module-security/tree/master/modules/fail2ban) from the [module-security
+repo](https://github.com/gruntwork-io/module-security), passing two custom parameters to it:
+
 
 ```
-gruntwork-install --module-name 'vault-ssh-helper' --repo 'https://github.com/gruntwork-io/script-modules' --tag '0.0.3' --module-param 'install-dir=/opt/vault-ssh-helper' --module-param 'owner=ubuntu'
+gruntwork-install --module-name `fail2ban` --repo `module-security` -module-param ban-time=3600
```

Comment:
These should be single quotes, not backticks... And the `bantime=3600` should be wrapped in single quotes too.

Reply:
@brikis98 Done